README.md

```diff
@@ -73,8 +73,19 @@ according to rules pr will be merged. Otherwise I ask changes. Just give me time
 
 Versioning
 ============
-Currently I just increment build number and version number itself doesn't have any logic in it.
-Maybe in the future will use Semver or similar. Not mandatory yet because haven't have need for breaking changes.
+We use [Semver versioning](https://semver.org/)
+
+```
+Given a version number MAJOR.MINOR.PATCH, increment the:
+
+MAJOR version when you make incompatible API changes,
```

Comment:
when incompatible API changes are made

Reply:
Hmm.. it's directly from Senver website but okay